portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,10 +26,38 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> comments;
+  
+
+  @Override
+  public void init() {
+      comments = new ArrayList<>();
+      comments.add("How did you create this gorgeous website");
+      comments.add("How did you join the SPS program");
+      comments.add("Sharing this website with some friends");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Amulya!</h1>");
+    String json = convertToJson(comments);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
   }
+
+  private String convertToJson(List<String> comments) {
+    String json = "{";
```

Comment:
If you are going to use String concatenation, I would prefer you use StringBuilder.

https://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java